src/views/TrackPositionAnimation/TrackPositionAnimationView.tsx

```diff
@@ -90,24 +93,6 @@ const usePositionFrames = (positions: number[][] | undefined, timestampStart: nu
     }, [positions, timestampStart, timestamps, headDirection, decodedData])
 }
 
-const setupAnimationStateDispatchFn = (animationStateDispatch: React.Dispatch<AnimationStateAction<PositionFrame>>) => {
```

Comment:
This stuff is relevant to every `FrameAnimation` and so has been moved to the component itself.